velero-plugin-for-aws/helpers.go

```diff
@@ -64,3 +65,30 @@ func IsValidS3URLScheme(s3URL string) bool {
 	}
 	return true
 }
+
+func CheckTags(awsTags string) error {
+	tags := strings.Split(awsTags, "&")
```

Comment:
Also, remove the blank spaces if there are any

Reply:
I could just fine a blank line. removed it.